source/fem-dem/ib_particles_dem.cc

```diff
@@ -520,31 +573,71 @@ IBParticlesDEM<dim>::calculate_pw_lubrication_force(
 {
   using numbers::PI;
 
+  struct DefaultDBL_MAX
+  {
+    double value = DBL_MAX;
+  };
+  struct DefaultUINT_MAX
+  {
+    int value = UINT_MAX;
+  };
+  enum lowest_floating_wall_indices
+  {
+    lowest_floating_wall_indices = 1000000
+  };
+
   // Loop over the particles
   for (auto &particle : dem_particles)
```

Comment:
I feel like there are repetitions there, are you sure you can't refactor some of the stuff in the previous routine to work also with this one?

Reply:
I move all the definitions in the .h, so they share all the definitions of the struct and of the enum.